pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.iabtcf</groupId>
     <artifactId>iabtcf-core</artifactId>
-    <version>1.0.3-alpha.0-SNAPSHOT</version>
+    <version>2.0.0-alpha.0-SNAPSHOT</version>
```

Comment:
is this being released?

Reply:
not sure what you mean by this question?

Reply:
I mean why we care about version if we haven't merged to master, or have we decided to merge now?

Reply:
i  just wanted to bump the version so that it agrees with the 2.XXX specification.